frontend/src/index.css

```
@@ -53,6 +53,8 @@
     --font-sans: "Inter", "Noto Sans SC", system-ui, sans-serif;
     --font-display: "Inter", "Noto Sans SC", system-ui, sans-serif;
     --font-mono: "JetBrains Mono", "Fira Code", "Noto Sans Mono", ui-monospace, monospace;
+    /* 衬线 — 仅项目大厅 (Darkroom) 用于编辑式电影感大标题 */
+    --font-editorial: "Instrument Serif", "Songti SC", "Noto Serif SC", Georgia, serif;
```

Stylelint (17.10.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)